mobile/Features/GettingStarted/Splash.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react';
+import {
+  Text, View, Pressable, StyleSheet, TouchableOpacity, Image,
+} from 'react-native';
+import IDMlogo from '../../assets/idontmindlogo.png';
+import styles from './SplashStyle';
+
+function Splash({ navigation }) {
+
+    const navigateToSplash = () => {
+        navigation.navigate('Terms');
+      };
+  
+  return (
```

Comment:
I fear that your eslint is not working? 😢
the indentation is a little strange 😢
besides indentation and formating, there's a couple other eslint issues that I'm getting, in particular with variables defined and not used.

Could you walk through the eslint guide again 😭 or if that truly doesnt work, then for this pr and in the future, could you buddy up with someone who does have eslint working for them to open up your files, save (autoformat), and point out the issues that exist?

if eslint worked for you last year (tcw), use this one:
https://www.notion.so/lablueprint/ESLint-b3426039ec824387b6602e598a844030